File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
    }while(dollars <= 0);
    int cent = round( 100 * dollars );
    int counter = 0;
    int changes[] = {25, 10, 5, 1};
    for (int i = 0; i < 4; i++)
    {
        if ( cent/changes[i] >= 1)
        {
            counter += cent/changes[i];
            cent -= cent/changes[i]*changes[i];
        }
    }
    printf("%i\n", counter);
    
}

File: pset6/cash/cash.py
while True:
    try:
        cash = float(input("Change owed: "))
    except:
        continue
    if cash > 0:
            break

cash *= 100
counter = 0
for i in [25, 10, 5, 1]:
    while cash >= i:
        counter += int(cash / i)
        cash %= i
print(counter)